packages/react-components/src/templates/__tests__/NetworkPeople.test.tsx

```diff
@@ -30,8 +37,12 @@ const props: ComponentProps<typeof NetworkPeople> = {
 };
 
 it('renders multiple people cards', () => {
-  const { queryAllByRole } = render(<NetworkPeople {...props} />);
+  const { queryAllByRole, queryAllByText } = render(
+    <NetworkPeople {...props} />,
+  );
   expect(
     queryAllByRole('heading').map(({ textContent }) => textContent),
   ).toEqual(['Person A', 'Person B']);
+
+  expect(queryAllByText('Barcelona Lab and Glasgow Lab')).toHaveLength(1);
```

Comment:
```suggestion
```

Reply:
The purpose of this test is to check the integration of the people card onto the page. Showing labs has already been tested in people card and therefore already covered